ext/drx/drx.c

```diff
@@ -556,16 +558,46 @@ drx_insert_counter_update(void *drcontext, instrlist_t *ilist, instr_t *where,
      * address being near this one, and add to reg1 instead of
      * taking 2 instrs to load it fresh.
      */
+    /* update counter */
```

Comment:
nit: In new comments use capitalization, punctuation, and complete sentences (you may see old comments not following this; it is a style change more recent than some old code)

Reply:
Got it.